rest/model/doc.go

```diff
@@ -1,35 +1,37 @@
 /*
-	Adding Models
+Package model maps database models to API models for the rest package.
```

Comment:
Same question as above about "API models for REST requests to the server" (or something along those lines).